lib/renovate.ts

```diff
@@ -13,7 +14,14 @@ proxy.bootstrap();
 
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 (async (): Promise<void> => {
-  process.exitCode = await globalWorker.start();
+  const tracer = getTracer();
+  process.exitCode = await tracer.startActiveSpan('run', async (span) => {
+    const exitCode = await globalWorker.start();
+    span.end();
+    return exitCode;
+  });
+  await getTracerProvider().shutdown(); //gracefully shutdown OpenTelemetry
```

Comment:
Do these changes result in effectively a "noop" if tracing is not enabled? i.e. doesn't slow down non-OTEL users?

Reply:
All auto instrumentations are not activated, but the manual ones are still active. 

I have changed it so that even that is `noop`ed.